src/components/Asset/AssetContent/MetaFull.tsx

```diff
@@ -5,10 +5,14 @@ import Publisher from '@shared/Publisher'
 import { useAsset } from '@context/Asset'
 import { getDummyWeb3 } from '@utils/web3'
 import { Asset, Datatoken, LoggerInstance } from '@oceanprotocol/lib'
+import { getPublisherNameOrOwner } from '@utils/ddo'
 
 export default function MetaFull({ ddo }: { ddo: Asset }): ReactElement {
   const [paymentCollector, setPaymentCollector] = useState<string>()
   const { isInPurgatory, assetState } = useAsset()
+  const complianceTypes = ddo?.metadata.additionalInformation.compliance
+  const isCompliant = complianceTypes?.length
```

Comment:
suggest to check for the actual `gaia-x` in the list

Reply:
Was done purposely. The flag is just to control the "(unverified)" part in the label. If another future compliance standard checker can verify the owner on another blockchain, that's fine too, it does not have to be Gaia-X perse.